### DISCUSSION TITLE
why the vairent styles not includes in the text

### DISCUSSION THREAD

**Discussion by karthicksakthi500** [callstack/react-native-paper#4569] [Q&A]
Why does React Native Paper focus on defining font sizes and typography properties directly in the Text component instead of using semantic variant names like error, info, title, subtitle, etc.? Wouldn't using these semantic variants make the component more reusable and consistent across the app, eliminating the need for creating additional components like HelperText for specific cases? Could you explain the reasoning behind this design choice, and if it’s possible to combine both approaches for better reusability?

**Comment by yashseth391** [comment]
Hi, i am also using paper , but my text color is not getting changed . can you tell which property is responsible for text color and how did you achieve it
